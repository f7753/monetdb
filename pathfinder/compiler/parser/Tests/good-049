#!/bin/bash

pf -s1 <<'EOT'
<a/>
EOT
echo $?
