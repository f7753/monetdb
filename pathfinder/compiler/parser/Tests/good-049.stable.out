stdout of test 'good-049` in directory 'compiler/parser` itself:


# 17:56:57 >  
# 17:56:57 >  Mtimeout -timeout 60 ./good-049 good-049 
# 17:56:57 >  

0

# 17:56:57 >  
# 17:56:57 >  Done.
# 17:56:57 >  

